tesseract/tesseract_geometry/include/tesseract_geometry/mesh_parser.h

```diff
@@ -258,11 +417,10 @@ std::vector<std::shared_ptr<T>> createMeshFromResource(tesseract_common::Resourc
 
   // Issue #38 fix: as part of the post-processing, we remove all other components in file but
   // the meshes, as anyway the resulting shapes:Mesh object just receives vertices and triangles.
+  // John Wason Dec 2020 - Load more information
```

Comment:
Same here.

Reply:
Also updated to use additional arguments